parts/k8s/windowskubeletfunc.ps1

```diff
@@ -451,6 +451,8 @@ if (`$hnsNetwork)
 # Restart Kubeproxy, which would wait, until the network is created
 Restart-Service Kubeproxy
 
+`$env:AZURE_ENVIRONMENT_FILEPATH="c:\k\azurestackcloud.json"

```

Comment:
Where do we set the values for c:\k\azurestackcloud.json?

Reply:
https://github.com/Azure/aks-engine/blob/master/parts/k8s/kuberneteswindowssetup.ps1#L206-L210